internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -88,15 +91,17 @@ const (
 type Opts struct {
 	cli.GlobalOpts
 	cli.WatchOpts
-	ClusterName    string
-	Provider       string
-	Region         string
-	IPAddresses    []string
-	IPAddress      string
-	DBUsername     string
-	DBUserPassword string
-	SkipMongosh    bool
-	store          store.AtlasClusterQuickStarter
+	ClusterName     string
+	Provider        string
+	Region          string
+	IPAddresses     []string
+	IPAddress       string
+	DBUsername      string
+	DBUserPassword  string
+	SampleDataJobID string
+	SkipSampleData  bool
```

Comment:
I feel like this should be opt-in rather than opt-out but feel free to ignore, that change may be too complecated

Reply:
yes, I agree with you but I am not sure how to achieve this result.